test/test-pages/embedded-videos/expected.html

```diff
@@ -10,20 +10,21 @@
 
         <p>At root</p>
         <iframe width="560" height="315" src="https://www.youtube.com/embed/LtOGa5M8AuU"
-        frameborder="0"></iframe>
+        frameborder="0" allowfullscreen=""></iframe>
         <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/LtOGa5M8AuU"
-        frameborder="0"></iframe>
-        <iframe src="https://player.vimeo.com/video/32246206?color=ffffff&title=0&byline=0&portrait=0"
-        width="500" height="281" frameborder="0"></iframe>
+        frameborder="0" allowfullscreen=""></iframe>
+        <iframe src="https://player.vimeo.com/video/32246206?color=ffffff&amp;title=0&amp;byline=0&amp;portrait=0"
```

Comment:
are the `&amp;` normal? I'd write the URL with and `&` directly (but maybe I'd be wrong)